tests/test_core.py

```
@@ -394,6 +394,27 @@ def test():
         with self.app.test_client() as c:
             self.assertRaises(RuntimeError, c.get, '/test')
 
+    def test_external_url(self):
+        """Test that external_url works, and is not overriding endpoint."""
+        Menu(self.app)
+        menu = self.app.extensions['menu']
+
+        url = 'https://python.org'
+
+        @self.app.route('/test')
+        def test():
+            return 'test'
+
+        item1 = menu.submenu('menuitem1')
+        item1.register(text='Test', external_url=url)
+        assert menu.submenu('menuitem1').url == url
+
+        item2 = menu.submenu('menuitem2')
+        item2.register(endpoint='test', text='Test', external_url=url)
```

Forbidden how? As in it should throw an exception?